### DISCUSSION TITLE
Provide .js file for download

### DISCUSSION THREAD

**Discussion by bluehazzard** [standardagents/arrow-js#76] [General]
TLDR: would it be possible to provide an arrow.js for download on the github release page?

Hi,
i am a complete noob in the whole node, typescript space.
I found arrow during the search of a reactive framework without the need of all this things.

in the documentation is this beautiful example with
```
<script type="module">
  import { reactive, html } from '/js/arrow.js';
  // Start your app here!
</script>
```
exactly what i want to do. But sadly i can not find any arrow.js for download and i do not know (honestly also do not want to learn) how to build the typescript project.
My use case is in a remote location with no internet, so using a CDN is out of question.
Would it be possible to provide an arrow.js for download on the github release page?

**Comment by s-p-b** [comment]
Just to add to this comment, I noticed that IIFE was recently implemented as a build option for arrow-js. I'd like to suggest that the arrow github / webpage make an IIFE version available for download without having to build it yourself - it would be helpful to newbies who are facing the very steep learning curve of js tooling, and would fit with the arrow-js idea of not needing build tools. 

(To give a bit of context, I am writing a plugin for the 3D modeling software Sketchup, which uses Chromium Embedded Framework to display custom dialogs. It loads everything with file://, and thus can't use ESM modules due to CORS restrictions, although everything else about it is completely modern).

**Reply by justin-schroeder** [reply]
Why not just use it directly from a CDN? For example: https://codepen.io/justin-schroeder/pen/KKeayMd

**Reply by ssilverman** [reply]
I too (I'm new at this) am having difficulty finding or building "arrow.js". I don't want to use it from a CDN because my devices that use it aren't connected to the Internet. One shouldn't rely on there being internet connectivity.

So I'll add my voice here too: How to build or acquire arrow.js?

I also want to say that this library is great for very small embedded applications. Even though I'm brand new at modern web development, I was able to convert my partially-server-side-rendered form into an HTML template in a day.

**Reply by Chaphasilor** [reply]
Just download https://esm.sh/@arrow-js/core@1.0.0-alpha.10/es2022/core.mjs and rename it to `index.mjs`. Alternatively install the npm package and extract it from there.
